### PR TITLE
Configure teletext fonts and resolution

### PR DESCRIPTION
**Pull request by persianpros**
Thanks to https://github.com/Fischreiher
It requires some changes in "config.py" too.
I didn't like https://github.com/openatv/enigma2/commit/13450b6d115e96bf2303c453f225f41a29dd109f#diff-a426b6df7279c1a44ef8a48173b058f5 for "Setup.py" so I didn't use it.

### CODE REVIEW

**Comment by littlesat** [comment]
I think this is too much

**Comment by persianpros** [comment]
Oh boy I spent hours on this ...

File: lib/python/Components/UsageConfig.py [hunk 1]
```diff
@@ -725,3 +755,84 @@ def preferredInstantRecordPath():
...
+	command = "sed -i -r '"
+	for f in tuxtxt2:
+		#replace keyword (%s) followed by any value ([-0-9]+) by that keyword \1 and the new value %d
+		command += "s|(%s)\s+([-0-9]+)|\\1 %d|;" % (f[0],f[1])
+	command += "' %s" % TUXTXT_CFG_FILE
+	for f in tuxtxt2:
+		#if keyword is not found in file, append keyword and value
+		command += " ; if ! grep -q '%s' %s ; then echo '%s %d' >> %s ; fi"  % (f[0],TUXTXT_CFG_FILE,f[0],f[1],TUXTXT_CFG_FILE)
+	try:
+		os.system(command)
```
Comment (athoik) [inline lib/python/Components/UsageConfig.py]: os.system? why? You can should execute using eConsoleAppContainer or using plain python to make those changes.

File: lib/python/Components/config.py [hunk 2]
```diff
@@ -127,11 +143,23 @@ def addNotifier(self, notifier, initial_call = True, immediate_feedback = True):
...
+			if extra_args:
+				notifier(self,extra_args)
+			else:
+				notifier(self)
 
 	def removeNotifier(self, notifier):
-		notifier in self.notifiers and self.notifiers.remove(notifier)
-		notifier in self.notifiers_final and self.notifiers_final.remove(notifier)
+		try:
+			del self.__notifiers[str(notifier)]
```
Comment (athoik) [inline lib/python/Components/config.py]: What exception do we get here?

File: lib/python/Components/config.py [hunk 3]
```diff
@@ -127,11 +143,23 @@ def addNotifier(self, notifier, initial_call = True, immediate_feedback = True):
...
+		except:
+			try:
+				del self.__notifiers_final[str(notifier)]
```
Comment (athoik) [inline lib/python/Components/config.py]: Trying again the same. How it will succeed now?

**Comment by littlesat** [comment]
I can imagine... Please ask first before create.... ;)

**Pull request closed without merging**